### PR TITLE
Fixes issue #116

### PR DESCRIPTION
**Pull request by bob2build**
## Pull Request Checklist

**Is this in reference to an existing issue?** Yes

#### General

- [*] Update Changelog following the conventions laid out [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)

- [*] Update README with any necessary configuration snippets

- [* ] Binstubs are created if needed

- [*] RuboCop passes

- [* ] Existing tests pass

#### Purpose
perform nil check before checking if zero

#### Known Compatibility Issues
none

### CODE REVIEW

**Comment by bob2build** [comment]
The tests always pass in local machine. However, seems to fail intermittently in travis. The intermittent failure is unrelated to the code change.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)
 
 ## [Unreleased]
+### Changed
```
Comment (majormoses) [inline CHANGELOG.md]: This should be `### Fixed` since it addresses a bug but I can fix that up post acceptance prior to release.

**Comment by majormoses** [comment]
a release build should be avilable shortly: https://rubygems.org/gems/sensu-plugins-rabbitmq/versions/7.0.1

**Comment by bob2build** [comment]
Thanks @majormoses

**Pull request merged**